### DISCUSSION TITLE
VcAnalyses分析工具直接在Cesium官方地形图上使用，通视分析正常，可视域分析全是绿色区域不正常。

### DISCUSSION THREAD

**Discussion by xujianhui1402** [zouyaoji/vue-cesium#190] [Q&A] [answered]
VcAnalyses分析工具直接在Cesium官方地形图上使用，通视分析正常，可视域分析全是绿色区域不正常。这是什么原因导致的呢？
![捕获](https://user-images.githubusercontent.com/67260794/162700917-816e3d43-453b-45f7-8370-af50528f58cb.PNG)

**Comment by zouyaoji** [comment, marked as answer]
目前只支持 3dtiles的哈，地形的不支持。文档有描述
![image](https://user-images.githubusercontent.com/24430116/162712197-ddc5c2f2-0dfa-4bab-a70d-f1b0d2a7d4ed.png)

**Reply by xujianhui1402** [reply]
感谢大佬，请问在V3.0.11 版本中，如何调整图元的位置高度呢？我发现3D Tiles模型与地面有一些偏差该如何调整呢？

**Reply by zouyaoji** [reply]
![image](https://user-images.githubusercontent.com/24430116/163080738-24100ed8-065e-4148-baa4-d477c1cc885d.png)

**Reply by xujianhui1402** [reply]
十分感谢，自己之前在官方文档中没有找到<显示代码>这块，一直是参考组件属性去摸索的。
